modules/database/sqlite/constants.php

```diff
@@ -6,12 +6,23 @@
  * @package performance-lab
  */
 
+// Backwards-compatibility.
+if ( ! defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) && defined( 'SQLITE_DB_DROPIN_VERSION' ) ) {
+	define( 'PERFLAB_SQLITE_DB_DROPIN_VERSION', SQLITE_DB_DROPIN_VERSION );
+}
+
 // Temporary - This will be in wp-config.php once SQLite is merged in Core.
-if ( ! defined( 'DATABASE_TYPE' ) ) {
-	if ( defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) ) {
-		define( 'DATABASE_TYPE', 'sqlite' );
+if ( ! defined( 'DB_ENGINE' ) ) {
+	if ( defined( 'SQLITE_DB_DROPIN_VERSION' ) ) {
```

Comment:
Based on the code above in line 10-12, we would need to use `PERFLAB_SQLITE_DB_DROPIN_VERSION` instead, as above you set that constant based on the other one. Which one is considered the source of truth? Given the standalone plugin uses `SQLITE_DB_DROPIN_VERSION`, we should probably use that.